app/src/components/adminConsole/SnappableManager.tsx

```diff
@@ -15,7 +19,7 @@ const SnappableManager = () => {
         (async () => {
             try {
                 const snappables = await getSnappables();
-                snappablesToCsvDownload(snappables);
+                csvContentToCsvDownload(snappablesToCsvContent(snappables));
```

Comment:
This looks a little clunky and leaks details about how the conversion process works.

I think you'd be better off with a `snappablesToCSVDownload` method and and a `snapsToCSVDownload` method that hides all these details.